<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.4.543">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Keller Lab - Scientific output from the lab</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../img/LogoBlack_square.png" rel="icon" type="image/png">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a href="../index.html" class="navbar-brand navbar-brand-logo">
    <img src="../img/small_logo3.png" alt="" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">Keller Lab</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../research/research.html"> 
<span class="menu-text">Research</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../people/people.html"> 
<span class="menu-text">People</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="../publications/publications.html" aria-current="page"> 
<span class="menu-text">Publications</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../photos/photos.html"> 
<span class="menu-text">Photos</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/KellerLabUVM"> <i class="bi bi-twitter-x" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://scholar.google.com/citations?user=cYPEa5gAAAAJ&amp;hl=en"> <i class="bi bi-google" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/stephenrkeller"> <i class="bi bi-github" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="mailto:srkeller@uvm.edu"> <i class="bi bi-envelope" role="img" aria-label="email">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
          <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#pre-prints-or-manuscripts-in-review" id="toc-pre-prints-or-manuscripts-in-review" class="nav-link active" data-scroll-target="#pre-prints-or-manuscripts-in-review">Pre-prints or manuscripts in review:</a></li>
  <li><a href="#published" id="toc-published" class="nav-link" data-scroll-target="#published">Published</a>
  <ul>
  <li><a href="#section" id="toc-section" class="nav-link" data-scroll-target="#section">2023</a></li>
  <li><a href="#section-1" id="toc-section-1" class="nav-link" data-scroll-target="#section-1">2022</a></li>
  <li><a href="#section-2" id="toc-section-2" class="nav-link" data-scroll-target="#section-2">2021</a></li>
  <li><a href="#section-3" id="toc-section-3" class="nav-link" data-scroll-target="#section-3">2020</a></li>
  <li><a href="#section-4" id="toc-section-4" class="nav-link" data-scroll-target="#section-4">2019</a></li>
  <li><a href="#section-5" id="toc-section-5" class="nav-link" data-scroll-target="#section-5">2018</a></li>
  <li><a href="#section-6" id="toc-section-6" class="nav-link" data-scroll-target="#section-6">2017</a></li>
  <li><a href="#section-7" id="toc-section-7" class="nav-link" data-scroll-target="#section-7">2016</a></li>
  <li><a href="#section-8" id="toc-section-8" class="nav-link" data-scroll-target="#section-8">2015</a></li>
  <li><a href="#section-9" id="toc-section-9" class="nav-link" data-scroll-target="#section-9">2014</a></li>
  <li><a href="#section-10" id="toc-section-10" class="nav-link" data-scroll-target="#section-10">2013</a></li>
  <li><a href="#section-11" id="toc-section-11" class="nav-link" data-scroll-target="#section-11">2012</a></li>
  <li><a href="#section-12" id="toc-section-12" class="nav-link" data-scroll-target="#section-12">2011</a></li>
  <li><a href="#section-13" id="toc-section-13" class="nav-link" data-scroll-target="#section-13">2006-2010</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content page-columns page-full" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Scientific output from the lab</h1>
<p class="subtitle lead">~ the results of our curiosity</p>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  


</header>


<hr>
<section id="pre-prints-or-manuscripts-in-review" class="level1 page-columns page-full">
<h1>Pre-prints or manuscripts in review:</h1>
<div class="column-default">
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ol type="1">
<li><p>Portlas, Z.M., J. Molofsky, L. Milbrath, and <strong><em>S.R. Keller</em></strong>. Preadaptation and post-introduction evolution in the invasive Centaurea jacea (Asteraceae) hybrid complex in response to drought stress. (In revision for American J. Botany)</p></li>
<li><p>Verrico, B.M. and <strong><em>S.R. Keller</em></strong>. Fine-scale adaptation in the presence of extensive gene flow in the coniferous tree red spruce (Picea rubens Sarg.). In revision for Evolution.</p></li>
<li><p>Capblancq, T., S. Lachmuth, M.C. Fitzpatrick, and <strong><em>S.R. Keller</em></strong>. From the last glacial maximum to the end of the 21th century: the journey of a montane tree species. In revision for Molecular Ecology.</p></li>
<li><p>Chhatre, V.E., K.C. Fetter, A.V. Gougherty, M.C. Fitzpatrick, R.Y. Soolanayakanahally, R.S. Zalensy, and <strong><em>S.R. Keller</em></strong>. Climatic niche predicts the landscape structure of locally adaptive standing genetic variation. In revision for Communications Biology. bioRxiv preprint: https://doi.org/10.1101/817411</p></li>
</ol>
</div>
</div>
</div>
</div>
<hr>
</section>
<section id="published" class="level1">
<h1>Published</h1>
<section id="section" class="level2">
<h2 class="anchored" data-anchor-id="section">2023</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ul>
<li><p>Fetter, K. and <strong><em>S.R. Keller</em></strong>, 2023. Admixture mapping and selection scans identify genomic regions associated with stomatal patterning and disease resistance in hybrid poplars. Ecology &amp; Evolution. https://onlinelibrary.wiley.com/doi/full/10.1002/ece3.10579</p></li>
<li><p>Lachmuth, S., T. Capblancq, A. Prakash, <strong><em>S.R. Keller</em></strong>, and M.C. Fitzpatrick. 2023. Novel genomic offset metrics account for local adaptation in climate suitability forecasts and inform assisted migration. Ecological Monographs. https://doi.org.10.1002/ecm.1593</p></li>
<li><p>Molofsky, J., D. Thom, <strong><em>S.R. Keller</em></strong>, and L.R. Milbrath. 2023.. Closely related invasive species may be controlled by the same demographic life stages. Neobiota 82:189-207, https://doi.org/10.3897/neobiota.82.95127</p></li>
<li><p>Capblancq,T., S. Lachmuth, M.C. Fitzpatrick and <strong><em>S.R. Keller</em></strong>. 2023. From common gardens to candidate genes: exploring local adaptation to climate in red spruce. New Phytologist https://doi.org/10.1111/nph.18465</p></li>
</ul>
</div>
</div>
</div>
</section>
<section id="section-1" class="level2">
<h2 class="anchored" data-anchor-id="section-1">2022</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ul>
<li><p>Molofsky, J., D.S. Park, D.M. Richardson, <strong><em>S.R. Keller</em></strong>, B. Beckage, J.R. Mandel, J.S. Boatwright and C. Hui. 2022. Optimal differentiation to the edge of trait space (EoTS). Evolutionary Ecology.</p></li>
<li><p>Prakash, A., S. DeYoung, S. Lachmuth, J.L. Adams, K. Johnsen, J.R. Butnor, D.M. Nelson, M.C. Fitzpatrick and <strong><em>S.R. Keller</em></strong>. 2022. Genotypic variation and plasticity in climate-adaptive traits after range expansion and fragmentation of red spruce (Picea rubens Sarg.) Philosophical Transactions of the Royal Society B, 377(1848).</p></li>
<li><p>A.J. Láruson, M.C. Fitzpatrick, <strong><em>S.R. Keller</em></strong>, B.C. Haller, K.E. Lotterhos. 2022. Seeing the forest for the trees: Assessing genetic offset predictions from gradient forest. Evolutionary Applications 15(3):403-416.</p></li>
<li><p>Gougherty, A.V., <strong><em>S.R. Keller</em></strong>, and M.C. Fitzpatrick. 2022. Do genetically informed distribution models improve range predictions in past climates? A case study with balsam poplar. Frontiers of Biogeography 14(3).</p></li>
</ul>
</div>
</div>
</div>
</section>
<section id="section-2" class="level2">
<h2 class="anchored" data-anchor-id="section-2">2021</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ul>
<li><p>An, X., K. Gao, Z. Chen, J. Li, X. Yang, J. Zhou, T. Guo, T. Zhao, S. Huang, D. Miao, W.U. Khan, P. Rao, M. Ye, B. Lei, W. Liao, J. Wang, L. Ji, Y. Li, B. Guo, N.S. Mustafa, S. Li, Q. Yun, <strong><em>S.R. Keller</em></strong>, J. Mao, R. Zhang, S.H. Strauss. 2021. High quality haplotype-resolved genome assemblies of Populus tomentosa Carr., a stabilized interspecific hybrid species widespread in Asia. Molecular Ecology Resources 22(2):786-802.</p></li>
<li><p>Capblancq, T., H. Munson, J.R. Butnor, and <strong><em>S.R. Keller</em></strong>. 2021. Genomic drivers of early-life fitness in Picea rubens. Conservation Genetics 22, 963–976</p></li>
<li><p>Fetter, K.C., D.M. Nelson, and <strong><em>S.R. Keller</em></strong>. 2021. Trade-offs and selection conflicts in hybrid poplars indicate the stomatal ratio as an important trait regulating disease resistance. Evolution.</p></li>
<li><p>Fitzpatrick, M.C., V.E. Chhatre, R. Soolanayakanahally, and <strong><em>S.R. Keller</em></strong>. 2021. Experimental support for genomic prediction of climate maladaptation using the machine learning approach Gradient Forests. Molecular Ecology Resources 21(8):2749-2765.</p></li>
<li><p>Gougherty, A.V., <strong><em>S.R. Keller</em></strong>, M.C. Fitzpatrick. 2021. Maladaptation, migration, and extirpation fuel climate change risk in a forest tree species. Nature Climate Change 11:166-171.</p></li>
</ul>
</div>
</div>
</div>
</section>
<section id="section-3" class="level2">
<h2 class="anchored" data-anchor-id="section-3">2020</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ul>
<li><p>Thibault, E., R. Soolanayakanahally, and <strong><em>S.R. Keller</em></strong>. 2020. Latitudinal clines in bud flush phenology reflect genetic variation in chilling requirements in balsam poplar, Populus balsamifera L. (Salicaceae). The American Journal of Botany 107:1597-1605.</p></li>
<li><p>Hildbrand, R.H., <strong><em>S.R. Keller</em></strong>, S.M. Laperriere, A.E. Santoro, J. Cessna, and R. Trott. 2020. Microbial communities can predict the ecological condition of headwater streams. PLoS One.</p></li>
<li><p>Capblancq, T., M.C. Fitzpatrick, R.A. Bay, M. Exposito-Alonso, and <strong><em>S.R. Keller</em></strong>. 2020. Genomic prediction of (mal)adaptation across current and future climatic landscapes. Annual Review of Ecology, Evolution, and Systematics, Vol. 51.</p></li>
<li><p>Capblancq, T., J. Butnor, S. DeYoung, E. Thibault, H. Munson, D.M. Nelson, M.C. Fitzpatrick, and <strong><em>S.R. Keller</em></strong>. 2020. Whole exome sequencing reveals a long-term decline in effective population size of red spruce (Picea rubens). Evolutionary Applications.</p></li>
<li><p>Laperriere, S.M., R.H. Hildbrand, <strong><em>S.R. Keller</em></strong>, R. Trott, and A.E. Santoro. 2020. Microbial indicators of watershed land use and headwater stream condition. Applied and Environmental Microbiology, 86:e00018-20.</p></li>
<li><p>Sun, Y., J. Ding, E. Siemann, and <strong><em>S.R. Keller</em></strong>. 2020. Biological control of invasive weeds under climate change: progress, challenges, and management implications. Current Opinion in Insect Science, 38:72-78.</p></li>
<li><p>Gougherty, A., V.E. Chhatre, <strong><em>S.R. Keller</em></strong>, and M.C. Fitzpatrick. 2020. Contemporary range position predicts the range-wide pattern of genetic diversity in balsam poplar (Populus balsamifera L.). Journal of Biogeography.</p></li>
<li><p>Verrico, B.M., J. Weiland, T.D. Perkins, B. Beckage, and <strong><em>S.R. Keller</em></strong>. 2020. Long-term monitoring reveals forest community change driven by atmospheric sulfate pollution and contemporary climate change. Diversity and Distributions, 26:270-283.</p></li>
</ul>
</div>
</div>
</div>
</section>
<section id="section-4" class="level2">
<h2 class="anchored" data-anchor-id="section-4">2019</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ul>
<li><p>Lachmuth, S., J. Molofsky, L. Milbrath, J. Suda, and <strong><em>S.R. Keller</em></strong>. 2019. Associations between genomic ancestry, genome size and capitula morphology in the invasive meadow knapweed hybrid complex (Centaurea × moncktonii C.E. Britton) in eastern North America. AoB Plants, 11:plz055.</p></li>
<li><p>Butnor, J.R., B.M. Verrico, K.J. Johnsen, C.A. Mayer, V. Vankus, and <strong><em>S.R. Keller</em></strong>. 2019. Phenotypic variation in climate-associated traits of red spruce (Picea rubens Sarg.) along elevation gradients in the southern Appalachian Mountains. Castanea 84:128-143.</p></li>
<li><p>Fetter, K.C., S. Eberhardt, R.S. Barclay, S. Wing, and <strong><em>S.R. Keller</em></strong>. 2019. StomataCounter: a neural network automatic stomata identification and counting. New Phytologist 223:1671-1681. (The definitive version is available at Wiley Interscience.)</p></li>
<li><p>Fetter, K. C. and A. Weakley. 2019. Reduced gene flow from mainland populations of Liriodendron tulipifera into the Florida peninsula promotes diversification. International Journal of Plant Sciences 180:253-269.</p></li>
<li><p>Schrieber, K., S. Wolf, C. Wypior, D. Hohlig, <strong><em>S.R. Keller</em></strong>, I. Hensen, and S. Lachmuth. 2019. Enemy release mitigates inbreeding depression in native and invasive Silene latifolia populations: experimental insight into the role of inbreeding × environment interactions in invasion success. Ecology and Evolution 9:3564-3576.</p></li>
<li><p>Hoogland, J.L., R. Trott, and <strong><em>S.R. Keller</em></strong>. 2019. Polyandry and polygyny in a social rodent: an integrative perspective based on social organization, copulations, and genetics. Frontiers in Ecology and Evolution.</p></li>
</ul>
</div>
</div>
</div>
</section>
<section id="section-5" class="level2">
<h2 class="anchored" data-anchor-id="section-5">2018</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ul>
<li><p>Gougherty, A.V., <strong><em>S.R. Keller</em></strong>, A. Kruger, C.D. Stylinski, A.J. Elmore, and M.C. Fitzpatrick. 2018. Estimating tree phenology from high frequency tree movement data. Agricultural and Forest Meteorology, 263:217-224.</p></li>
<li><p>Chhatre, V.E., L.M. Evans, S.P. DiFazio, and <strong><em>S.R. Keller</em></strong>. 2018. Adaptive introgression and maintenance of a trispecies hybrid complex in range-edge populations of Populus. Molecular Ecology 27:4820-4838.</p></li>
<li><p>Fitzpatrick, M.C., <strong><em>S.R. Keller</em></strong>, and K.E. Lotterhos. 2018. Comment on “Genomic signals of selection predict climate-driven population declines in a migratory bird.” Science 361, eaat7279.</p></li>
<li><p>Butnor, J.R., B.M. Verrico, V. Vankus, and <strong><em>S.R. Keller</em></strong>. 2018. Ethanol exposure can inhibit red spruce (Picea rubens Sarg.) germination. Seed Science and Technology, 46:259-265.</p></li>
</ul>
</div>
</div>
</div>
</section>
<section id="section-6" class="level2">
<h2 class="anchored" data-anchor-id="section-6">2017</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ul>
<li><p><strong><em>S.R. Keller</em></strong>, V.E. Chhatre, and M.C. Fitzpatrick. 2017. Influence of range position on locally adaptive gene-environment associations in Populus flowering time genes. Journal of Heredity, 109:47-58.</p></li>
<li><p>Palacio-Lopez, K., <strong><em>S.R. Keller</em></strong>, J.M. Molofsky. 2017. Genomic admixture between locally adapted populations of Arabidopsis thaliana (Mouse ear cress): Evidence of optimal genetic outcrossing distance. Journal of Heredity, 109:38-46.</p></li>
<li><p>Fetter, K.C., P. Gugger, and <strong><em>S.R. Keller</em></strong>. 2017. Landscape genomics in angiosperm trees: model systems at the foundations and forefront of the field. Comparative and Evolutionary Genomics of Angiosperm Trees. Andrew Groover and Quentin Cronk, eds.&nbsp;Springer, ISBN 978-3-319-49329-9.</p></li>
<li><p><strong><em>S.R. Keller</em></strong>, R. Hilderbrand, M.K. Shank, and M. Potapova. 2017. Environmental DNA genetic monitoring of the nuisance freshwater diatom, Didymospheniageminata, in eastern North American streams. Diversity and Distributions, 23:381-393. — see also the﻿ article in The Scientist featuring our eDNA work on Didymo﻿</p></li>
<li><p>Colautti, R.I., J.M. Alexander, K.M. Dglugosch, <strong><em>S.R. Keller</em></strong>, and S.E. Sultan. 2017. Invasions and extinctions: through the looking glass of evolutionary ecology. Philosophical Transactions of the Royal Society B, 372(1712):20160031. ​</p></li>
<li><p><strong><em>S.R. Keller</em></strong> 2017. A genetic assessment of the population health and connectivity of a keystone species in high elevation Appalachian forest ecosystems: red spruce (Picea rubens Sarg.). Final report to the West Virginia Department of Natural Resources (not peer-reviewed). July 20, 2017.</p></li>
</ul>
</div>
</div>
</div>
</section>
<section id="section-7" class="level2">
<h2 class="anchored" data-anchor-id="section-7">2016</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ul>
<li><p>Sebasky, M., <strong><em>S.R. Keller</em></strong>, and D.R. Taylor. 2016. Investigating past range dynamics for a weed of cultivation, Silene vulgaris. Ecology and Evolution, 6:4800-4811.</p></li>
<li><p>Pylant, C.L., D.M. Nelson, M.C. Fitzpatrick, J.E. Gates, and <strong><em>S.R. Keller</em></strong>. 2016. Geographic origins and population genetics of bats killed at wind-energy facilities. Ecological Applications, 26:1381.</p></li>
<li><p>Kazyak, D.C., R.H. Hilderbrand, T.L. King, <strong><em>S.R. Keller</em></strong>, and V.E. Chhatre. 2016. Hiding in plain sight: a case for cryptic metapopulations in brook trout. PLoS ONE 11(1): e0146295.</p></li>
</ul>
</div>
</div>
</div>
</section>
<section id="section-8" class="level2">
<h2 class="anchored" data-anchor-id="section-8">2015</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ul>
<li><p>Jaramillo-Correa, J.P., J. Prunier, A. Vazquez-Lobo, <strong><em>S.R. Keller</em></strong>, A. Moreno-Letelier. 2015. Molecular signatures of adaptation and selection in forest trees. Advances in Botanical Research, Vol. 74, 265-306.</p></li>
<li><p>Fitzpatrick, M.C. and <strong><em>S.R. Keller</em></strong>. 2015. Ecological genomics meets community-level modeling of biodiversity: Mapping the genomic landscape of current and future environmental adaptation. Ecology Letters 18:1-16.</p></li>
<li><p>Kazyak, D.C., R.H. Hilderbrand, <strong><em>S.R. Keller</em></strong>, M.C. Colaw, A.E. Holloway, R.P. Morgan III, and T.L. King. 2015. Spatial structure of morphological and neutral genetic variation in Brook Trout (Salvelinus fontinalis). Trans. Amer. Fisheries Soc. 144:480-490.</p></li>
</ul>
</div>
</div>
</div>
</section>
<section id="section-9" class="level2">
<h2 class="anchored" data-anchor-id="section-9">2014</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ul>
<li><p>Pylant, C.L., D.M. Nelson, and <strong><em>S.R. Keller</em></strong>. 2014. Stable hydrogen isotopes record the summering grounds of eastern red bats (Lasiurus borealis). PeerJ 2:e629.</p></li>
<li><p>Molofsky, J., <strong><em>S.R. Keller</em></strong>, S. Lavergne, M. Kaproth, and M. Eppinga. 2014. Human-aided admixture may fuel ecosystem transformation during biological invasions: theoretical and experimental evidence. Ecology and Evolution 4: 899-910.</p></li>
<li><p><strong><em>S.R. Keller</em></strong>, P. D. Fields, A. E. Berardi, and D. R. Taylor. 2014. Recent admixture generates heterozygosity-fitness correlations during the range expansion of an invading species. Journal of Evolutionary Biology 27:616-627.</p></li>
</ul>
</div>
</div>
</div>
</section>
<section id="section-10" class="level2">
<h2 class="anchored" data-anchor-id="section-10">2013</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ul>
<li><p><strong><em>S.R. Keller</em></strong>, R. Trott, C. Pylant, and D. Nelson. 2013. Genome-wide microsatellite marker development from next-generation sequencing of two non-model bat species impacted by wind turbine mortality: Lasiurus borealis and L. cinereus (Vespertilionidae). Molecular Ecology Resources. 14:435-436.</p></li>
<li><p>Olson, M.S., N. Levsen, R. Soolanayakanahally, R. Guy, W. Schroeder, <strong><em>S.R. Keller</em></strong>, and P. Tiffin. 2013. The adaptive potential of the dominant boreal forest tree Populus balsamifera L. to phenology requirements in a warmer global climate. Mol. Ecol. 22:1214-1230.</p></li>
</ul>
</div>
</div>
</div>
</section>
<section id="section-11" class="level2">
<h2 class="anchored" data-anchor-id="section-11">2012</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ul>
<li><p><strong><em>S.R. Keller</em></strong>, K. J. Gilbert, P. D. Fields, and D. R. Taylor. 2012. Bayesian inference of a complex invasion history revealed by nuclear and chloroplast genetic diversity in the colonizing plant, Silene latifolia. Mol. Ecol. 21:4721-4734.</p></li>
<li><p><strong><em>S.R. Keller</em></strong>, N. Levsen, M.S. Olson,and P. Tiffin. 2012. Local adaptation in the flowering time genetic network of balsam poplar, Populus balsamifera L. Mol. Biol. Evol. 29:3143-3152.</p></li>
</ul>
</div>
</div>
</div>
</section>
<section id="section-12" class="level2">
<h2 class="anchored" data-anchor-id="section-12">2011</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ul>
<li><p>Sloan, D.B., <strong><em>S.R. Keller</em></strong>, A.E. Berardi, B.J. Sanderson, J.F. Karpovich, and D.R. Taylor. 2011. De novo transcriptome assembly and polymorphism detection in the flowering plant Silene vulgaris (Caryophyllaceae). Mol. Ecol. Res. 12:333-343.</p></li>
<li><p><strong><em>S.R. Keller</em></strong>, N. Levsen, P. K. Ingvarsson, M. S. Olson, and P. Tiffin. 2011. Local selection across a latitudinal gradient shapes nucleotide diversity in balsam poplar, Populus balsamifera L. Genetics.</p></li>
<li><p><strong><em>S.R. Keller</em></strong>, R. Soolanayakanahally, R. Guy, S. Silim, M. Olson, and P. Tiffin. 2011. Climate-driven local adaptation in ecophysiology and phenology in balsam poplar, Populus balsamifera L. (Salicaceae). American Journal of Botany 98:99-108.</p></li>
<li><p>Jenkins, C.E. and <strong><em>S.R. Keller</em></strong>. 2011. A phylogenetic comparative study of preadaptation for invasiveness in the flowering plant genus Silene (Caryophyllaceae). Biological Invasions 13:1471-1486.</p></li>
</ul>
</div>
</div>
</div>
</section>
<section id="section-13" class="level2">
<h2 class="anchored" data-anchor-id="section-13">2006-2010</h2>
<div>

</div>
<div class="quarto-layout-panel" data-layout="[ 1, 99 ]">
<div class="quarto-layout-row">
<div id="first-column" class="quarto-layout-cell" style="flex-basis: 1.0%;justify-content: flex-start;">

</div>
<div id="second-column" class="quarto-layout-cell" style="flex-basis: 99.0%;justify-content: flex-start;">
<ul>
<li><p><strong><em>S.R. Keller</em></strong>, and D.R. Taylor. 2010. Genomic admixture increases fitness during a biological invasion. Journal of Evolutionary Biology 23:1720-1731.</p></li>
<li><p><strong><em>S.R. Keller</em></strong>, M.S. Olson, S. Silim, W. Schroeder, and P. Tiffin. 2010. Genomic diversity, population structure and migration following rapid range expansion in the Balsam Poplar, Populus balsamifera. Molecular Ecology, 19:1212-1226. — see also the ‘Perspectives’ piece in this issue of Mol. Ecol. by Martin Lascoux and Remy Petit</p></li>
<li><p>Fields, P.D., <strong><em>S.R. Keller</em></strong>, P.K. Ingvarsson, A. Pederson, and D.R. Taylor. 2010. Isolation and characterization of polymorphic microsatellite loci in the white campion, Silene latifolia (Caryophyllaceae). Molecular Ecology Resources, 10:232-238.</p></li>
<li><p><strong><em>S.R. Keller</em></strong>, D.R. Sowell, M. Neiman, L.M. Wolfe, and D.R. Taylor. 2009. Adaptation and colonization history affect the evolution of clines in two introduced species. New Phytologist, 183:678-690.</p></li>
<li><p><strong><em>S.R. Keller</em></strong>, and D.R. Taylor. 2008. History, chance, and adaptation during biological invasion: separating stochastic phenotypic evolution from response to selection. Ecology Letters, 11: 852-866.</p></li>
<li><p>Sloan, D.B., C.M. Barr, M.S. Olson, <strong><em>S.R. Keller</em></strong>, and D.R. Taylor. 2008. Evolutionary rate variation at multiple levels of biological organization in plant mitochondrial DNA. Molecular Biology and Evolution, 25:243-246.</p></li>
<li><p>Etterson, J.R., <strong><em>S.R. Keller</em></strong>, and L.F. Galloway. 2007. Epistatic and cytonuclear interactions govern outbreeding depression in the autotetraploid Campanulastrum americanum. Evolution, 61:2671-2683.</p></li>
<li><p>Barr, C.M., <strong><em>S.R. Keller</em></strong>, P.K. Ingvarsson, D.B. Sloan, and D.R. Taylor. 2007. Variation in mutation rate and polymorphism among mitochondrial genes of Silene vulgaris. Molecular Biology and Evolution, 24:1783-1791.</p></li>
<li><p>Taylor, D.R. and <strong><em>S.R. Keller</em></strong>. 2007. Historical range expansion determines the phylogenetic diversity introduced during contemporary species invasion. Evolution, 61:334-345.</p></li>
<li><p><strong><em>S.R. Keller</em></strong>and K.E. Schwaegerle. 2006. Maternal sex and mate relatedness affect offspring quality in the gynodioecious Silene acaulis. Journal of Evolutionary Biology, 19:1128-1138.</p></li>
</ul>
</div>
</div>
</div>
<center>
<p><img src="../img/UVM_logo_green.png" style="height: 25%; width: 25%; object-fit: contain" onclick="window.open('https://www.uvm.edu', 'blank');"></p>
<center>


</center></center></section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      // TODO in 1.5, we should make sure this works without a callout special case
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">
<p><span class="faux-block">© Copyright 2024, Keller lab | Ecological Genomics</span></p>
</div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>




</body></html>