<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.29">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Adaptation and response to environmental change – Keller Lab</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../img/LogoBlack_square.png" rel="icon" type="image/png">
<script src="../site_libs/quarto-html/quarto.js" type="module"></script>
<script src="../site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting-9cfdc87e16b2bf3d6786c761657ef46f.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap-6e6a04bf1c2ee053824bde8dc3e63b3c.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script src="../site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="../site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="../site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "textbox",
  "limit": 50,
  "keyboard-shortcut": [
    "f",
    "/",
    "s"
  ],
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-text-placeholder": "",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>


</head>

<body class="nav-fixed quarto-light">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg " data-bs-theme="dark">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container mx-auto">
    <a href="../index.html" class="navbar-brand navbar-brand-logo">
    <img src="../img/small_logo3.png" alt="" class="navbar-logo">
    </a>
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">Keller Lab</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" role="menu" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link active" href="../research/research.html" aria-current="page"> 
<span class="menu-text">Research</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../people/people.html"> 
<span class="menu-text">People</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../publications/publications.html"> 
<span class="menu-text">Publications</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../photos/photos.html"> 
<span class="menu-text">Photos</span></a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://twitter.com/KellerLabUVM"> <i class="bi bi-twitter-x" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://scholar.google.com/citations?user=cYPEa5gAAAAJ&amp;hl=en"> <i class="bi bi-google" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/stephenrkeller"> <i class="bi bi-github" role="img">
</i> 
<span class="menu-text"></span></a>
  </li>  
  <li class="nav-item compact">
    <a class="nav-link" href="mailto:srkeller@uvm.edu"> <i class="bi bi-envelope" role="img" aria-label="email">
</i> 
<span class="menu-text"></span></a>
  </li>  
</ul>
          </div> <!-- /navcollapse -->
            <div class="quarto-navbar-tools">
</div>
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-full page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#research-themes" id="toc-research-themes" class="nav-link active" data-scroll-target="#research-themes">Research themes</a>
  <ul>
  <li><a href="#hybridization-and-adaptive-introgression" id="toc-hybridization-and-adaptive-introgression" class="nav-link" data-scroll-target="#hybridization-and-adaptive-introgression">Hybridization and Adaptive Introgression</a></li>
  <li><a href="#genomic-forecasting" id="toc-genomic-forecasting" class="nav-link" data-scroll-target="#genomic-forecasting">Genomic Forecasting</a></li>
  <li><a href="#invasions" id="toc-invasions" class="nav-link" data-scroll-target="#invasions">Invasions</a></li>
  </ul></li>
  <li><a href="#research-systems" id="toc-research-systems" class="nav-link" data-scroll-target="#research-systems">Research systems</a>
  <ul>
  <li><a href="#poplar" id="toc-poplar" class="nav-link" data-scroll-target="#poplar">Poplar</a></li>
  <li><a href="#red-spruce" id="toc-red-spruce" class="nav-link" data-scroll-target="#red-spruce">Red spruce</a></li>
  <li><a href="#northeast-alpine" id="toc-northeast-alpine" class="nav-link" data-scroll-target="#northeast-alpine">Northeast Alpine</a></li>
  </ul></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content page-columns page-full column-body" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Adaptation and response to environmental change</h1>
<p class="subtitle lead">~ the questions that drive us</p>
</div>



<div class="quarto-title-meta column-body">

    
  
    
  </div>
  


</header>


<div class="column-screen center">
<p><img src="website_banner_research_transparent_background3.png" class="nolightbox img-fluid"></p>
</div>
<section id="research-themes" class="level1 page-columns page-full">
<h1>Research themes</h1>
<section id="hybridization-and-adaptive-introgression" class="level2 page-columns page-full">
<h2 class="anchored" data-anchor-id="hybridization-and-adaptive-introgression">Hybridization and Adaptive Introgression</h2>
<p><strong>Status:</strong> Ongoing<br>
<strong>Publications</strong></p>
<div class="column-screen page-columns page-full">

<div class="no-row-height column-margin column-container"><div class="">
<div class="margin-photo">
<p>Poplar hybridization in the Pacific Northwest.<br>
<a href="poplar_introgression.jpg" class="lightbox" data-gallery="Introgression" title="Poplar hybridization in the Pacific Northwest."><img src="poplar_introgression.jpg" class="img-fluid" alt="Poplar hybridization in the Pacific Northwest."></a></p>
</div>
</div></div><p>While we often think of species as being reproductively isolated from one another, this is not always the case. As a consequence of weak reproductive barriers, members of different species can interbreed in nature. When hybrids between two species are fertile, hybrids may then interbreed with members of the original species in a process known as backcrossing. Multiple generations of hybridization and subsequent backcrossing can facilitate the exchange of genetic material between parental species, a process known as introgression. Much of the research conducted in the Keller lab aims to understand if this exchange of genetic material is an adaptive process. For example, does genetic variation that increases tolerance to environmental stress in one species play the same role when it is introgressed into another species?</p>
<p>One current area of interest in the Keller lab is the susceptibility of hybrid poplars to infection by rust fungi in the genus Melampsora. One overarching question in the lab that this work addresses is the relationship between hybridization and local adaptation. The potential roles of ancestry and adaptation to local climatic variables as indicators of disease susceptibility in common garden experiments are being investigated.</p>
</div>
</section>
<section id="genomic-forecasting" class="level2 page-columns page-full">
<h2 class="anchored" data-anchor-id="genomic-forecasting">Genomic Forecasting</h2>
<p><strong>Status:</strong> Ongoing<br>
<strong>Publications</strong></p>
<div class="column-screen page-columns page-full">

<div class="no-row-height column-margin column-container"><div class="">
<div class="margin-photo">
<p>Genomic forecasting workflow. <a href="genomic_forecasting.png" class="lightbox" data-gallery="forecasting" title="Genomic forecasting workflow."><img src="genomic_forecasting.png" class="img-fluid" alt="Genomic forecasting workflow."></a></p>
</div>
</div></div><p>Forecasting the responses of populations to climate change is a complex but important issue to understand as we face the potential for widespread climate maladaptation in the coming decades. This hits especially hard for long-lived, sessile organisms like forest trees, whose long-generation times and limited dispersal potential constrain the ability to adapt or track shifting climate. Recently, there has been increasing interest in combining genomics with climate models to forecast the short-term impacts of climate change on population adaptation. Our lab, in close collaboration with <a href="https://mfitzpatrick.al.umces.edu/Site/Welcome.html">Matt Fitzaptrick’s lab</a> has been developing computational approaches to predict short-term disruption of climate adaptation, known as “genomic offset” modelling. Since first introducing the concept and statistical framework for genomic offset in 2015 (Fitzpatrick and Keller 2015), we have worked to refine and extend the scope and application of offset models to predicting climate change maladaptation. This has involved simulation testing of genomic offset predictions, in collaboration with <a href="https://sites.google.com/site/katielotterhos/home">Katie Lotterhos’ lab</a>, as well as testing the sensitivity of offset models to model choices such as the genomic loci used and the climate variables and future projection scenarios (Lachmuth et al.&nbsp;2023, Frontiers).</p>
<p>In addition to building and refining the computational pipeline of genomic offset for forecasting climate change responses, our lab has been working on empirical testing of offset predictions, using a space-for-time approach that evaluates the effect of transferring genotypes from their source climate into one or more common garden sites, and predicting growth and fitness-traits from genomic offsets calculated between the source and garden site climates. Using poplar trees as a study system, we provided the first such common garden test of offset predictions, showing a strong negative response of growth with increasing genomic offset (Fitzpatrick et al.&nbsp;2021). We have recently shown a similar result in red spruce (Picea rubens) across 3 seedling common gardens (Lachmuth et al.&nbsp;2023; Eco Mon), as well as in a 60 year old red spruce provenance trial in northern New Hampshire (Verrico et al., in prep.).</p>
</div>
</section>
<section id="invasions" class="level2">
<h2 class="anchored" data-anchor-id="invasions">Invasions</h2>
<p><strong>Status:</strong> Completed</p>
<p>The movement of species into new habitats allows populations that were previously geographically isolated to reproduce with each other and create novel genotypes through hybridization. Through environmental selection, novel genotypes can result in phenotypes that increase hybrid fitness and promote biological invasions. Furthermore, hybridization between genetically distinct species can result in changes to the genome size or species ploidy, which can alter the reproduction rate and promote invasion success. The rapid evolution of hybrid species in the introduced range can lead to a drastic shift in the stable state dynamics of the ecosystem, transforming the community assembly.</p>
<p>Past research has largely focused on the effects of hybridization in Centaura × moncktonii (<em>C. jacea × C. nigra</em>), a plant invasive to North America known as Japanese knapweed. Using a combination of field collections at hybrid zones in eastern North America and the Pacific northwest.</p>
<p>An assessment of the genome size, degree of introgression, and species traits reveals that hybrid individuals have smaller genomes and capitula traits that promote reproductive success, indicating that admixture of <em>C. jacea</em> and <em>C. nigra</em> can generate invasive genotypes that are environmentally selected for.</p>
</section>
</section>
<section id="research-systems" class="level1 page-columns page-full">
<h1>Research systems</h1>
<section id="poplar" class="level2 page-columns page-full">
<h2 class="anchored" data-anchor-id="poplar">Poplar</h2>
<p><strong>Status:</strong> Ongoing<br>
<strong>Publications</strong></p>
<div class="column-screen page-columns page-full">

<div class="no-row-height column-margin column-container"><div class="">
<div class="margin-photo">
<p>Stomatal patterning of Poplar leaf.<br>
<a href="./poplar/img1.png" class="lightbox" data-gallery="Introgression"><img src="./poplar/img1.png" class="img-fluid"></a></p>
</div>
<div class="margin-photo">
<p>Walk through the common garden. <a href="./poplar/common_garden_walk.jpg" class="lightbox" data-gallery="Introgression"><img src="./poplar/common_garden_walk.jpg" class="img-fluid"></a></p>
</div>
</div></div><p>Poplar is an established model system for tree genomics, and P. trichocarpa was the first tree to have its full genome sequenced. Whereas poplar populations harbor rich genetic diversity, and hybridization between poplar species is both common in the wild and a key aspect of poplar breeding programs, conventional approaches to genomic analysis, which generally depend on a reference genome from a single individual, are poorly suited to poplar genomics. In the Keller Lab, we have gone to great lengths to overcome this issue by leveraging the growing field of pan-genomics.</p>
<p>In the Keller Lab, research on hybridized poplar trees investigates the genetic and physiological trade-offs associated with increased photosynthetic efficiency and susceptibility to disease. F1 hybrids often display heterosis, resulting in traits such as increased stomatal density and larger stomatal pore sizes, which significantly enhance photosynthetic rates and industrial yield. These traits are relevant for applications such as biofuel and the paper production. However, this increased productivity is associated with a trade-off in reduced disease resistance to Melampsora, a rust pathogen commonly affecting poplar species. The current study aims to find the genetic basis of these trade-offs and to identify hybrids that retain high industrial yields while mitigating susceptibility to disease.</p>
<p>Leaf samples from F1 hybrids were collected at the UVM Horticulture Farm and analyzed for photosynthetic efficiency. Stomatal patterning, a key factor influencing both photosynthesis and pathogen susceptibility, is assessed through microscopic analysis of leaf imprints at 40x magnification. In addition to phenotypic analysis, RNAseq data from young hybrid leaves was utilized to identify genes involved in stomatal patterning and disease resistance. Field assessments of Melampsora rust severity were conducted by visually rating the extent of infection in each hybrid. This disease severity data, in combination with stomatal trait analysis and sequencing data, will provide insights into the genetic determinants of rust resistance. By correlating phenotypic and genotypic data, the study aims to identify specific genetic markers that can inform future breeding efforts to develop hybrid poplar varieties that balance enhanced productivity with improved disease resistance.</p>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Poplar photos
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<div class="quarto-layout-panel" data-layout-nrow="1">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: flex-start;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="./poplar/common_garden_walk.jpg" class="lightbox" data-gallery="poplar" title="Walking through the poplar garden at Vermont"><img src="./poplar/common_garden_walk.jpg" class="img-fluid figure-img" alt="Walking through the poplar garden at Vermont"></a></p>
<figcaption>Walking through the poplar garden at Vermont</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: flex-start;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="./poplar/poplar_leaf_miner.jpg" class="lightbox" data-gallery="poplar" title="Poplar leaf miner"><img src="./poplar/poplar_leaf_miner.jpg" class="img-fluid figure-img" alt="Poplar leaf miner"></a></p>
<figcaption>Poplar leaf miner</figcaption>
</figure>
</div>
</div>
<div class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: flex-start;">
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="./poplar/rusty.jpg" class="lightbox" data-gallery="poplar" title="Rusty poplars"><img src="./poplar/rusty.jpg" class="img-fluid figure-img" alt="Rusty poplars"></a></p>
<figcaption>Rusty poplars</figcaption>
</figure>
</div>
</div>
</div>
</div>
</div>
</div>
</div>
</section>
<section id="red-spruce" class="level2">
<h2 class="anchored" data-anchor-id="red-spruce">Red spruce</h2>
<p><strong>Status:</strong> Ongoing<br>
<strong>Publications</strong></p>
<p><strong>Rangewide population genetics and local adaptation</strong></p>
<p>Red spruce (Picea rubens) is an ecologically important conifer adapted to cool, moist, high-elevation sites mainly in the northeastern U.S. and southeastern Canada, but the southern tail of the range extends down into fragmented mountaintop populations in the Central and Southern Appalachians. Red spruce provides timber, critical habitat for rare and endemic wildlife species, and recreational value to humans. Our lab has explored how population size, range limits, and genetic diversity have impacted local adaptation to climate across red spruce’s range. [Anoob’s Proc Soc B paper, Thibaut’s papers]</p>
<p><strong>Hybridization and adaptive introgression</strong></p>
<p>For long-lived species with low genetic diversity, introgression—the transfer of genetic material from one lineage into the genome of another through repeated backcrossing—can be a powerful source of adaptive variation in the face of climate change. Red spruce hybridizes with its sister species black spruce (Picea mariana), and most red spruce trees found in the overlap between the two species’ ranges derive significant portions of their genome from past hybridization with black spruce. Recent work by our lab suggests that introgression from black spruce plays a key role in supporting red spruce’s capacity to adapt to climate gradients across its range [Anoob’s adaptive introgression paper here]. We use a combination of spatial modeling and landscape and population genomics to characterize the spatial and temporal natural history of hybridization, understand the role of selection in preserving introgressed regions of the genome, and determine under what conditions this introgression may be adaptive. We are also establishing a provenance trial of red and black spruce hybrids, which will serve as a long-term resource for linking the genomics of hybridization with adaptive traits.</p>
<p><strong>Applied conservation</strong></p>
<p>One of our lab’s major goals is to disseminate the findings of the scientific research to the broader community to inform conservation and restoration efforts on the ground. Red spruce is a species of conservation concern due to high climate change vulnerability and is the focus of several major restoration initiatives, notably the Central Appalachian Red Spruce Restoration Initiative (CASRI) and the Southern Appalachian Red Spruce Restoration Initiative (SASRI), which have planted tens of thousands of seedlings supported by The Nature Conservancy.</p>
<p><strong>Spruce orchard project</strong></p>
<p>In a project funded by the USDA-FS Forest Health Protection program, we are applying the genomic offset model, a method for forecasting genomic maladaptation to climate change, to establish a red spruce seed orchard in the Green Mountain National Forest. Forest managers have traditionally applied a “local is best” framework when choosing where to source seeds for restoration plantings. However, the genotypes currently found in a particular region may not be the ones best adapted to survive the future climate conditions that the region is likely to experience by the end of the century. We have sourced red spruce cuttings from trees that capture a diversity of genotypes predicted by our genomic offset models to be well-adapted to future climates in New England. Once the grafted seedlings become reproductive in 5-7 years, they will produce significant quantities of future climate-adapted seed to support reforestation and assisted migration plantings, a key need expressed by conservation partners in a workshop on red spruce hosted by our lab group in 2021.</p>
<p><strong>Assisted migration project</strong></p>
<p>We are working with our partners at The Nature Conservancy, the U.S. Fish and Wildlife Service, and the Vermont Agency of Natural Resources to collect seedling performance data from a network of recently established assisted migration field trials.</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-2-contents" aria-controls="callout-2" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Red spruce restoration in collaboration with CASRI and TNC
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-2" class="callout-2-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<p>Kathryn Shallows from TNC approached the Keller lab in 2019 regarding red spruce restoration towards the southern range edge of its distribution, mainly in Maryland, Virginia and West Virginia. This led to a larger partnership that culminated in genomic assisted seed source selection for the restoration initiative at these sites. Instead of opting for a single source restoration initiative, we used a multiple seed source approach to improve the genetic diversity of the restoration stands to withstand the changing climate and environmental stressors. This source selection was carried out in the light of research done by Keller lab using the exome capture data and the common gardens set up at Vermont, Maryland and North Carolina. Three to four sources combinations per restoration site were selected for high genetic diversity and low genetic load.</p>
<div class="line-block"></div>
<p>Capblancq et al.&nbsp;(2021) has observed that early-life fitness of the red spruce had strong positive association with genetic diversity and negative association with genetic load, especially for the southern range edge of the red spruce distribution range. These seed source combinations were used as a recommendation list to Dave Seville (CASRI) to assess the seed production situation on the ground and raise them in the nursery for the restoration initiative. In 2021, TNC planted 58,000 red spruce seedlings at the restoration sites. Keller lab then carried out the restoration monitoring of these restoration sites in 2022, a whole year after the seedlings had been planted in the ground to assess the success of genomic assisted restoration of red spruce. Using this information, Prakash et al.&nbsp;(in prep) found that there were no “Super Seed Source” that outperformed other seed sources and the combination of seed sources outperformed any single source for the restoration success.</p>
<div class="quarto-figure quarto-figure-center">
<figure class="figure">
<p><a href="TNC_conservation_genetics.png" class="lightbox" data-gallery="red-spruce" title="Genomic assisted red spruce restoration at Maryland, West-Viginia and Virginia, USA."><img src="TNC_conservation_genetics.png" class="img-fluid figure-img" alt="Genomic assisted red spruce restoration at Maryland, West-Viginia and Virginia, USA."></a></p>
<figcaption>Genomic assisted red spruce restoration at Maryland, West-Viginia and Virginia, USA.</figcaption>
</figure>
</div>
</div>
</div>
</div>
</section>
<section id="northeast-alpine" class="level2 page-columns page-full">
<h2 class="anchored" data-anchor-id="northeast-alpine">Northeast Alpine</h2>
<p><strong>Status:</strong> Ongoing</p>

<div class="no-row-height column-margin column-container"><div class="">
<div class="margin-photo">
<p><a href="./alpine/img1.png" class="lightbox" data-gallery="alpine"><img src="./alpine/img1.png" class="img-fluid"></a></p>
</div>
<div class="margin-photo">
<p><a href="./alpine/img2.jpg" class="lightbox" data-gallery="alpine"><img src="./alpine/img2.jpg" class="img-fluid"></a></p>
</div>
<div class="margin-photo">
<p><a href="./alpine/img6.jpg" class="lightbox" data-gallery="alpine"><img src="./alpine/img6.jpg" class="img-fluid"></a></p>
</div>
</div></div><p>The northeastern North American alpine habitat, home to regionally rare and vulnerable species, occurs at high elevation in spatially isolated mountain regions. Since glacial retreat ~13,000 years ago, communities of alpine plants have shifted as highly connected tundra landscapes have retreated into today’s fragmented “sky islands” of various sizes. We have yet to understand the underlying adaptive mechanisms responsible for the persistence of alpine plants despite dramatic shifts in climate, though the genomic diversity of these plant communities have likely been shaped by this biogeographic history.</p>
<p>The potential impacts of climate change on northeastern alpine plants is a topic of considerable conservation concern. Little is known about levels of adaptive capacity within alpine plant populations, their degree of resilience or susceptibility to changes in climate, and how these might vary from peak to peak across the northeast. Thus, there is a need for studies investigating how the fitness of alpine plants may be driven by local adaptation or genetic variability in trait plasticity (GxE) to determine the vulnerability or resilience of these communities to climate change.</p>
<p>This project, funded by the <a href="https://www.nsf.gov/awardsearch/showAward?AWD_ID=2326021">National Science Foundation Biodiversity on a Changing Planet</a> program, links paleoecology, community ecology, trait ecology, population genomics, and physical geography to identify the mechanisms underlying the persistence of tundra relics in Northeast alpine zones since deglaciation. By integrating these approaches, we aim to develop a process-based understanding of how Northeast alpine plant communities assembled, and how these communities persisted through past climate changes as an integration of species, traits, and genes. Our research area focuses on four alpine regions in Northeastern US: the Adirondacks (NY), the Green Mountains (VT), the White Mountains (NH), and Katahdin (ME).</p>
<p><strong>Collaborators:</strong></p>
<ul>
<li><a href="https://www.meireleslab.org/#">Miereles Lab (University of Maine, Orono)</a><br>
</li>
<li><a href="https://www.ecoevomatics.org/">Rominger Lab (University of Hawaiʻi at Mānoa)</a><br>
</li>
<li><a href="https://jacquelyngill.wordpress.com/">BEAST Lab (University of Maine, Orono)</a><br>
</li>
<li><a href="https://www.cmmackenzie.com/">Dr.&nbsp;Caitlin McDonough MacKenzie</a><br>
</li>
<li><a href="https://isaacovercast.github.io/">Dr.&nbsp;Isaac Overcast</a></li>
</ul>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-3-contents" aria-controls="callout-3" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class="callout-icon"></i>
</div>
<div class="callout-title-container flex-fill">
Alpine photos
</div>
<div class="callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end"><i class="callout-toggle"></i></div>
</div>
<div id="callout-3" class="callout-3-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<div class="quarto-layout-panel" data-layout-nrow="1">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: center;">
<p><a href="./alpine/img3.jpg" class="lightbox" data-gallery="alpine"><img src="./alpine/img3.jpg" class="img-fluid"></a></p>
</div>
<div class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: center;">
<p><a href="./alpine/img4.jpg" class="lightbox" data-gallery="alpine"><img src="./alpine/img4.jpg" class="img-fluid"></a></p>
</div>
<div class="quarto-layout-cell" style="flex-basis: 33.3%;justify-content: center;">
<p><a href="./alpine/img5.jpg" class="lightbox" data-gallery="alpine"><img src="./alpine/img5.jpg" class="img-fluid"></a></p>
</div>
</div>
</div>
</div>
</div>
</div>
<center>
<p><img src="../img/UVM_logo_green.png" style="height: 25%; width: 25%; object-fit: contain" onclick="window.open('https://www.uvm.edu', 'blank');"></p>
<center>


</center></center></section>
</section>

</main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->
<footer class="footer">
  <div class="nav-footer">
    <div class="nav-footer-left">
      &nbsp;
    </div>   
    <div class="nav-footer-center">
<span class="faux-block"> © Copyright <span id="year"></span>, Keller lab | Ecological Genomics </span>
<script>
  document.getElementById("year").textContent = new Date().getFullYear();
</script>
</div>
    <div class="nav-footer-right">
      &nbsp;
    </div>
  </div>
</footer>
<script>var lightboxQuarto = GLightbox({"closeEffect":"zoom","descPosition":"bottom","loop":false,"openEffect":"zoom","selector":".lightbox"});
(function() {
  let previousOnload = window.onload;
  window.onload = () => {
    if (previousOnload) {
      previousOnload();
    }
    lightboxQuarto.on('slide_before_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      const href = trigger.getAttribute('href');
      if (href !== null) {
        const imgEl = window.document.querySelector(`a[href="${href}"] img`);
        if (imgEl !== null) {
          const srcAttr = imgEl.getAttribute("src");
          if (srcAttr && srcAttr.startsWith("data:")) {
            slideConfig.href = srcAttr;
          }
        }
      } 
    });
  
    lightboxQuarto.on('slide_after_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(slideNode);
      }
    });
  
  };
  
})();
          </script>




</body></html>